src/modules/tasks.js

```diff
@@ -139,3 +144,23 @@ export function addTask(description) {
   saveTasks();
   renderTasks();
 }
+
+const updateTaskIndexes = () => {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+};
+
+export function clearCompletedTasks() {
+  for (let i = tasks.length - 1; i >= 0; i -= 1) {
+    if (tasks[i].completed) {
+      tasks.splice(i, 1);
```

Comment:
- [ ] I would implore you to use the required method which is a filter

![image](https://github.com/victormc13/To-Do-List/assets/86200356/1c60309f-67fc-48cd-9e6c-bd5e7b328222)

The filter is best to use in this case. It would take something like:
`tasks.filter((task) => !task.completed)` so it would filter out the tasks that have been completed false. Do you understand? If not, let me know please, for a better explanation.